### ISSUE TITLE
i18n in sendEmailOnMessage.js not working

### ISSUE THREAD

**Issue by localguru** [RocketChat/Rocket.Chat#7313]
Rocket.Chat Version: 0.57.0-develop and 0.56.0

Sending emails on message are not localized in packages/rocketchat-lib/server/lib/sendEmailOnMessage.js. My global locale and user profile is set to "de", but I receive email with "en" localization, e.g. Subject "[Rocket.Chat] You have been direct messaged by testuser". Also "Offline_Link_Message" is not translated and set to "GO TO MESSAGE". System locale on Ubuntu 16.04 LTS: 

```
LANG=de_DE.UTF-8
LANGUAGE=
LC_CTYPE="de_DE.UTF-8"
LC_NUMERIC="de_DE.UTF-8"
LC_TIME="de_DE.UTF-8"
LC_COLLATE="de_DE.UTF-8"
LC_MONETARY="de_DE.UTF-8"
LC_MESSAGES="de_DE.UTF-8"
LC_PAPER="de_DE.UTF-8"
LC_NAME="de_DE.UTF-8"
LC_ADDRESS="de_DE.UTF-8"
LC_TELEPHONE="de_DE.UTF-8"
LC_MEASUREMENT="de_DE.UTF-8"
LC_IDENTIFICATION="de_DE.UTF-8"
LC_ALL=
```

see #4078 

Should be something like this, but how do I get my current locale:
```
const message = TAPi18n.__('Offline_Link_Message', {}, 'de');
```

**Comment by localguru** [comment]
Also the timezone of the Date: mail header is not correct, set to +0000 in case of CET

**Comment by theorenck** [comment]
@MarcosSpessatto can you take a look at this one and see if it's still a problem?

**Comment by pierre-lehnen-rc** [comment]
Fixed by #13452
